src/main/java/ch/njol/skript/expressions/ExprEntityOwner.java

```diff
@@ -0,0 +1,102 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.bukkitutil.UUIDUtils;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.entity.Tameable;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+@Name("Entity Owner")
+@Description({
+	"The owner of a tameable entity (i.e. horse or wolf) or a dropped item.",
+	"Getting the owner of a dropped item will only return the uuid of the owner",
+	"Setting the owner of a dropped item means only that entity or player can pick it up.",
+	"Dropping an item does not automatically make the entity or player the owner."
+})
+@Example("""
+		set owner of last spawned wolf to player
+		if the owner of last spawned wolf is player:
+	"""
+)
+@Example("""
+		set dropped item owner of last dropped item to player
+		if the dropped item owner of last dropped item is uuid of player:
+	"""
+)
+@Since("2.5, INSERT VERSION (dropped items)")
+public class ExprEntityOwner extends SimplePropertyExpression<Entity, Object> {
+
+	static {
+		Skript.registerExpression(ExprEntityOwner.class, Object.class, ExpressionType.PROPERTY,
+			"[the] (owner|tamer) of %livingentities%",
+			"%livingentities%'[s] (owner|tamer)",
+			"[the] [dropped item] owner of %itementities%",
+			"%itementities%'[s] [dropped item] owner");
```

Comment:
```suggestion
			"[the] [dropped] item owner of %itementities%",
			"%itementities%'[s] [dropped] item owner");
```
The omitted version will always be obscured by the original pattern, so no need to make it optional.
I think the pattern's rough but i don't think there's really a better option here with all the inconsistent API.

Also, what's the point of the different patterns if you don't treat them differently? The point of making them different is so the first can deal with entities, while the second deals with uuids. Also see comment on thrower's syntax.

Reply:
So as per your requested changes on `ExprItemThrower` I should make this 
```
[the] uuid of [the] [dropped] item owner [of %itementities%]
[the] [dropped] item owner's uuid [of %itementities%]
``` 
?

and if so, should I just pull it out into it's own expression?

Reply:
Yeah, probably. It's really awkward trying to deal with the same concept that's using two different types so it's probably best to separate them to make it as explicit as possible.